javascript/stimulus_reflex.js

```diff
@@ -14,17 +14,21 @@ import {
 
 // A reference to the Stimulus application registered with: StimulusReflex.initialize
 //
-let app
+let stimulusApplication
```

Comment:
Are you renaming this for clarity, or is there a functional change?

Reply:
Both clarity and consistency given that we're now also tracking `actionCableConsumer`